micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/MetricsConfiguration.groovy

```diff
@@ -68,4 +70,11 @@ class MetricsConfiguration {
     MetricRegistry metricRegistry(MetricPathProvider metricPathProvider) {
         return new PathPrependingMetricRegistry(metricPathProvider)
     }
+
+    @Bean
+    public HystrixCodaHaleMetricsPublisher hystrixCodaHaleMetricsPublisher(MetricRegistry metricRegistry) {
+        def publisher = new HystrixCodaHaleMetricsPublisher(metricRegistry)
```

Comment:
It is a three-liner with a very long class name used on the right. I don't know if it is much more readable as:
```
HystrixCodaHaleMetricsPublisher publisher = new HystrixCodaHaleMetricsPublisher(metricRegistry)
HystrixPlugins.instance.registerMetricsPublisher(publisher)
return publisher
```